README.md

```diff
@@ -1,38 +1,16 @@
 # obal - packaging wrapper using ansible
 
-## Updating a Package from Upstream
+`obal` is an ansible wrapper with a set of ansible playbooks to ease maintanance of packaging repositories like [`foreman-packaging`](https://github.com/theforeman/foreman-packaging) and [`pulp-packaging`](https://github.com/pulp/pulp-packaging).
```

Comment:
General comment, since Ansible is a proper noun we should capitalize it anywhere we reference the project itself.